src/css/archwizard.scss

```diff
@@ -0,0 +1,70 @@
+// color definitions
+$wz-color-default: #E6E6E6 !default;
+$wz-color-current: #808080 !default;
+$wz-color-done: #339933 !default;
+$wz-color-optional: #38ef38 !default;
+$wz-color-editing: #FF0000 !default;
+
+// dot definitions
+$wz-dot-border-width: 2px !default;
+
+// small definitions
+$wz-small-dot-width: 14px !default;
+$wz-small-dot-height: 14px !default;
+
+// big definitions
+$wz-big-dot-width: 50px !default;
+$wz-big-dot-height: 50px !default;
+
+// extra distance between the bottom of the dots and the baseline texts
+$wz-dot-baseline-distance: 0px !default;

```

Comment:
If I remember correctly, doesn't thin variable depend on the layout (horizontal vs. vertical)?

Reply:
It was used in both, but was 0px in both cases:

```
@dot-baseline-distance: 0px
```

We can provide separate `$wz-dot-baseline-distance-horizontal` and `$wz-dot-baseline-distance-vertical`? Do you think it is needed?

Reply:
Totally agree. Removed the variable in 029cd280.